app/routes/wow/full-scan/SmallTable.tsx

```diff
@@ -65,14 +65,28 @@ function SmallTable<Type>({
 
   const columnHelper = createColumnHelper<Type>()
 
+  const returnLocaleString = (value: any) => {
```

Comment:
A better name would be to `toLocaleString`
`return` is developer talk which is a anti-pattern when naming things